test/components/whats-new.spec.js

```diff
@@ -0,0 +1,79 @@
+import Modal from '../../src/components/modal.vue';
+import WhatsNew from '../../src/components/whats-new.vue';
+
+import testData from '../data';
+import { load } from '../util/http';
+import { mockLogin } from '../util/session';
+
+describe('WhatsNew modal', () => {
+  describe('shows modal', () => {
+    it('shows modal to admin with 0 projects', async () => {
+      mockLogin({ createdAt: '2025-01-01' });
+      const app = await load('/', { root: false });
+      const baseModal = app.findComponent(WhatsNew).findComponent(Modal);
+      baseModal.exists().should.be.true;
+      baseModal.props().state.should.be.true;
+    });
+
+    it('shows modal to admin with more than 1 project', async () => {
+      mockLogin({ createdAt: '2025-01-01' });
+      testData.extendedProjects.createPast(1);
+      const app = await load('/', { root: false });
+      const baseModal = app.findComponent(WhatsNew).findComponent(Modal);
+      baseModal.exists().should.be.true;
+      baseModal.props().state.should.be.true;
+    });
+
+    it('shows modal to project manager with 1 project', async () => {
+      mockLogin({ role: 'manager' });
```

Comment:
The `role` here is the sitewide role.

```suggestion
      mockLogin({ role: 'none' });
```